### PR TITLE
fix(pytorch): runnable method collision

### PR DESCRIPTION
**Pull request by bojiang**

### CODE REVIEW

File: src/bentoml/_internal/frameworks/common/pytorch.py [hunk 1]
```diff
@@ -46,7 +46,7 @@
 
 def partial_class(
     cls: t.Type[PytorchModelRunnable], *args: t.Any, **kwargs: t.Any
-) -> type:
+) -> t.Type[PytorchModelRunnable]:
```
Comment (aarnphm) [inline src/bentoml/_internal/frameworks/common/pytorch.py:49]:
```suggestion
) -> type[PytorchModelRunnable]:
```

File: src/bentoml/_internal/frameworks/pytorch.py [hunk 2]
```diff
@@ -197,18 +197,19 @@ def get_runnable(bento_model: Model):
 
     partial_kwargs: t.Dict[str, t.Any] = bento_model.info.options.partial_kwargs  # type: ignore
 
+    runnable_class: t.Type[PytorchModelRunnable] = partial_class(
```
Comment (aarnphm) [inline src/bentoml/_internal/frameworks/pytorch.py:200]:
```suggestion
    runnable_class: type[PytorchModelRunnable] = partial_class(
```

File: src/bentoml/_internal/frameworks/common/pytorch.py [hunk 3]
```diff
@@ -46,7 +46,7 @@
 
 def partial_class(
     cls: t.Type[PytorchModelRunnable], *args: t.Any, **kwargs: t.Any
```
Comment (bojiang) [inline src/bentoml/_internal/frameworks/common/pytorch.py:48]:
```suggestion
    cls: type[PytorchModelRunnable], *args: t.Any, **kwargs: t.Any
```

**Review by bojiang — COMMENTED** [review]
done

**Review by aarnphm — APPROVED** [review]

**Review by larme — COMMENTED** [review]
Tested and it solved current problem. Should we also add this change to `bentoml.torchscript`?

**Pull request merged**